content/python/get_virtual_guest_usage_records.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "CPU and Memory usage in Virtual servers"
+description: "How the portal page retrieves the memory and cpu usage"
+date: "2019-04-10"
+classes:
+    - "SoftLayer_Virtual_Guest"
+    - "SoftLayer_Metric_Tracking_Object"
+tags:
+    - "metric"
+    - "metrics"
+    - "virtual server"
```

Comment:
lets go with `virtualguest` as the tag. (so remove `virtual guest`)
and remove the `metrics` tag, no need for the pluralization.

Reply:
Done